Tests/Fixtures/Generated/Strings/objc-m/localizable-headerName.m

```diff
@@ -0,0 +1,71 @@
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+#import "L10N.h"
+
+@interface BundleToken : NSObject
+@end
+
+@implementation BundleToken
+@end
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wformat-security"
+
+static NSString* tr(NSString *tableName, NSString *key, ...) {
+    va_list args;
+    va_start(args, key);
+
+    NSBundle *bundle = [NSBundle bundleForClass:BundleToken.class];
+    NSString *format = [bundle localizedStringForKey:key value:@"" table:tableName];
+    NSString *result = [[NSString alloc] initWithFormat:format arguments:args];
+    return result;
```

Comment:
I think we are supposed to call `va_end(args)` once we are finished with the variadic args?

Reply:
good catch! i'll fix that.